datalad/distributed/export_archive_ora.py

```diff
@@ -103,6 +137,9 @@ def __call__(
         else:
             archive.parent.mkdir(exist_ok=True, parents=True)
 
+        if isinstance(froms, str):
+            froms = [froms]
```

Comment:
```suggestion
        froms = ensure_list(froms)
```

plus the `from datalad.utils import ensure_list`